README.md

```diff
@@ -1,17 +1,17 @@
-# commerce-sdk
-
-Salesforce Commerce Node.js SDK
+# Salesforce Commerce Node.js SDK
 
 [![CircleCI][circleci-image]][circleci-url]
 
-## What is This?
+## What is This
 
-A mono repo containing packages that generate the Salesforce Commerce SDK. Read more about the SDK [here](./packages/generator/README.md) and the tools in this toolkit in the [Packages](#packages) section.
+A monorepo containing packages that generate the Salesforce Commerce SDK. Read more about the SDK [here](./packages/generator/README.md) and the tools in this toolkit in the [Packages](#packages) section.
```

Comment:
I think we can get rid of "and the tools in this toolkit in the [Packages](#packages) section." since we do not of have any tools other than generator